src/program/config/data_format/data_format.lua

```diff
@@ -0,0 +1,135 @@
+-- Use of this source code is governed by the Apache 2.0 license; see COPYING.
+module(..., package.seeall)
+
+local lib = require("core.lib")
+local schema = require("lib.yang.schema")
+local yang_data = require("lib.yang.data")
+local usage = require("program.config.data_format.README_inc")
+
+-- Number of spaces a tab should consist of when indenting config.
+local tab_spaces = 4
+
+local function show_usage(status)
+   print(require("program.config.data_format.README_inc"))
+   main.exit(status)
+end
+
+local function parse_args(args)
+   local handlers = {}
+   handlers.h = function() show_usage(0) end
+   args = lib.dogetopt(args, handlers, "h", {help="h"})
+   if #args <= 0 then show_usage(1) end
+   return unpack(args)
+end
+
+function run(args)
+   function print_level(level, ...)
```

Comment:
Because you don't prefix these with "local", all these functions (except the ones of the form `function foo.bar`) are getting added to the top level.  But actually I think that's where they should be -- `run` should handle parsing arguments and exiting successfully but other things should be done outside run.  Would you mind making that refactor please?